activation/post.go

```diff
@@ -489,6 +491,34 @@ func (mgr *PostSetupManager) GenerateProof(ctx context.Context, challenge []byte
 	return p, m, nil
 }
 
+func (mgr *PostSetupManager) VerifyProof(ctx context.Context, proof *types.Post, metadata *types.PostMetadata, options ...verifying.OptionFunc) error {
```

Comment:
what's the rationale not reusing the existing code path in `Validator.Post()`?

Reply:
but if codepath diverge, the benefit is lost. miners still only found out proof is invalid after poet round end.
would it be cleaner to pass the validator to atx builder in node.go?

Reply:
I agree with @countvonzero there shouldn't be two code paths to verify a Post Proof

Reply:
Changed to use the `nipostValidator` interface.